Dockerfile

```diff
@@ -73,9 +73,9 @@ ENTRYPOINT ["python3", "-m", "vllm.entrypoints.api_server"]
 
 # openai api server alternative
 FROM vllm-base AS vllm-openai
-# install additional dependencies for openai api server
+# install additional dependencies for openai api server, and mixtral
 RUN --mount=type=cache,target=/root/.cache/pip \
-    pip install accelerate fschat
+    pip install accelerate megablocks
```

Comment:
fschat is needed for openai server.  I think megablocks should be added to the requirements.txt instead. This way it will be used in both vllm-openai and vllm

Reply:
fschat is no longer needed for openai server. But i'm indeed closing this in favor of #2029.

Reply:
BTW, why do we need `accelerate`?

Reply:
I don't recall the reason. But I will experiment with removing it in a separate PR.